src/options.ts

```diff
@@ -295,7 +295,7 @@ export const getOptions = () => {
     ),
     micromambaVersion: parseOrUndefined(
       'micromamba-version',
-      z.union([z.literal('latest'), z.string().regex(/^\d+\.\d+\.\d+-\d+$/)]),
+      z.union([z.literal('latest'), z.string().regex(/^\d+\.\d+\.\d+(?:\.(?:rc|alpha|beta)\d+)?-\d+$/)]),
```

Comment:
Should we add `dev` as well since it was added [here](https://github.com/mamba-org/mamba/pull/3639/files#diff-5d68e15117de5e835e8f3224e12c8b44b41b3d761aeb327fb9e1694c8a2365dfR23)?